unicef/juniper-portfolio
# Story 179 add wallet modal

Frontend work for the add wallet modal.

To Test:
1. From the wallet screen, click 'Add New Wallet'
![image](https://user-images.githubusercontent.com/6285466/83570991-f786bf00-a4f4-11ea-921f-8c54d825c006.png)

2. Verify modal matches screens
![image](https://user-images.githubusercontent.com/6285466/83571175-4cc2d080-a4f5-11ea-958a-bbc4d1fc4e83.png)

Wireframes:
https://app.zeplin.io/project/5e729a0ffe1440ba52a35ccd/screen/5ed132ab083da4b9bd946786
https://app.zeplin.io/project/5e729a0ffe1440ba52a35ccd/screen/5ed132ac083da4b9bd9467b0
https://app.zeplin.io/project/5e729a0ffe1440ba52a35ccd/screen/5ed132ac30a0789b54d1b7e4

3. Toggle Multisig
4. Add multisig owners

Callout: No 'Remove Multisig Owner' in designs. I will prioritize setting up a staging env in rancher tomorrow so that we can deploy and let design in. This will also give me a chance to setup the branches and environments per our standards.
